oio/cli/admin/item_locate.py

```diff
@@ -0,0 +1,393 @@
+# Copyright (C) 2019 OpenIO SAS, as part of OpenIO SDS
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from itertools import chain
+from urllib import quote
+from urlparse import urlparse
+
+from cliff import lister
+
+from oio.common import exceptions
+from oio.common.fullpath import encode_fullpath
+from oio.common.utils import cid_from_name
+from oio.cli.admin.common import ContainerCommandMixin, ObjectCommandMixin
+
+
+DUMMY_SERVICE = {'addr': None, 'score': 0, 'tags': {}}
+
+
+class ItemLocateCommand(lister.Lister):
+    """
+    A command to display which service is in charge of hosting an item.
+    """
+
+    columns = ('Type', 'Item', 'Service Id', 'Addr', 'Location', 'Status')
```

Comment:
We could add `Volume`.

Reply:
We could replace `Status` with 2 columns (`Up` and `Score`)

Reply:
What would we display in the `Volume` column?

I don't like the idea of splitting `Status`: currently it displays either the status (in the form of a boolean and a score) or the error that occurred during the check. Where would we display the error then?